target/linux/ath79/dts/qca9558_engenius_emr3000.dts

```diff
@@ -0,0 +1,157 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "qca955x.dtsi"
+
+/ {
+	model = "EnGenius EMR3000";
+	compatible = "engenius,emr3000", "qca,qca9558";
+
+	aliases {
+		led-boot = &status_white;
+		led-failsafe = &status_white;
+		led-running = &status_white;
+		led-upgrade = &status_white;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		status_white: status_white {
+			label = "emr3000:white:status";
```

Comment:
Model prefix "emr3000:" can be dropped from LEDs now, leaving just "white:status" etc.